.github/actions/display-jest-test-summary/README.md

```diff
@@ -0,0 +1,28 @@
+# Display Jest Test Summary
+Action to output results of jest tests to Github summary
```

Comment:
Do you have a screengrab of what it looks like in the summary? I'd like to see it!

Reply:
@alexisc-kainos please answer above

Reply:
Hi @AnnaDodson I have added a screenshot to the README, this is an example of the MOT Recalls e2e tests